shared/actions/teams.js

```diff
@@ -1194,7 +1194,6 @@ const teamAvatarUpdated = (_, action) => {
 
 const teamChangedByName = (state, action) => {
   const {teamName} = action.payload.params
-  logger.info(`Got teamChanged for ${teamName} from service`)
```

Comment:
There's a lot of logging that's going on that doesn't seem relevant. This looks like a debug log that stayed around, so I killed it.

Reply:
I think in general the logs for service <-> GUI calls are very sparse, this log line is the only way to know this came in.

Reply:
Is this really helpful for us on a daily basis?

Reply:
Well, when we get a report, we hopefully should be able to tell what happened by looking at the logs. I think the action logs alone give a very fuzzy picture of what happened.

(Come to think of it, since engine notifs recently started dispatching actions, we would see that in the logs. This still gives the teamname which the notif log doesn't.)